environment.yml

```diff
@@ -2,29 +2,29 @@ name: evidential
 channels:
   - conda-forge
 dependencies:
-  - python=3.8
-  - numpy
+  - python =3.8
+  - numpy ~=1.23.5
```

Comment:
What does the `~=` symbol mean in the environment file? I would prefer not to specify library versions unless absolutely necessary (e.g., tensorflow probability only works with specific versions of tensorflow and each version of tensorflow has a limited supported range of versions with numpy) because it can cause conda dependency conflicts down the line.

Reply:
It specifies a range of version types. I removed them for now and will test the new environment.